data/functions/calendar.sql

```diff
@@ -178,8 +178,7 @@ returns text as $$
                 array[
                     expand_office_description(office_sought),
                     report_type,
-                    'Due Today',
-                    'States:',
+                    'Due Today. States:',
```

Comment:
How would you feel about making this say `is Due Today in the Following States:` ?